Documentation/5.4.1/README.rst

```diff
@@ -20,6 +20,7 @@
 .. ..  10/23/18    5.3.3     Jennifer - DOC-2112
 .. ..  10/24/18    5.3.3     Jennifer - DOC-2105 - AVRS
 .. ..  01/22/19    5.4.1     DOC-2202 - Edit file in github
+.. ..  03/04/19    5.4.1     PROD-9386 - Adding Smart NIC Documentation
```

Comment:
I put a bunch of additional name changes into https://github.com/psairam/nuage-ospdirector/pull/6 - please take a look and merge into this PR or let's discuss.

Reply:
Sure Scott, I will pull those changes into this PR.